cdhweb/people/tests.py

```diff
@@ -419,7 +431,7 @@ def test_staff_list(self):
 
         response = self.client.get(reverse('people:staff'))
         # person should only appear once even if they have multiple positions
-        assert len(response.context['current']) == 1
+        assert response.context['current'].filter(user__username='staff').count() == 1
```

Comment:
Why the username filter? Does the test content now include more than one person? Asking because I can't tell if this is a useful test / if it's actually checking anything.

Reply:
that's exactly why, yeah. a lot of these tests are unfortunately pretty brittle because they're sharing a lot of state via the big fixture...it's not great.